cms/djangoapps/contentstore/views/preview.py

```diff
@@ -223,7 +222,8 @@ def _preview_module_system(request, descriptor, field_data):
             "teams_configuration": TeamsConfigurationService(),
             "sandbox": SandboxService(contentstore=contentstore, course_id=course_id),
             "cache": CacheService(cache),
-            'replace_urls': replace_url_service
+            'replace_urls': replace_url_service,
+            'error_descriptor_class': ErrorBlock
```

Comment:
Alternatively, we can create a ErrorDescriptorService with `__init__(descriptor_class)` and with `get_error_descriptor(descriptor, error_message)` method, should we go with that?